connaisseur/validators/notaryv2/notaryv2_validator.py

```diff
@@ -1,6 +1,10 @@
 from connaisseur.validators.interface import ValidatorInterface
+from connaisseur.image import Image
 
 
 class NotaryV2Validator(ValidatorInterface):
     def __init__(self, name: str, host: str, **kwargs):
+        raise NotImplementedError
```

Comment:
This could set `self.name = name`, instead of raising, and instead raise in `.validate()`. You can then call `super().__init__(name, host, **kwargs)` in the specific implementations.

Also, do *all* conceivable validators need a `host` argument?